landing_page/spec/views/users/show.html.erb_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe "users/show.html.erb" do
+
+    context "when the user has data" do 
+
+        it "displays a users profile correctly" do
+
```

Comment:
can remove this blank line

Reply:
Resolved in commit bcc1c2e3f03b36170702a98979328c0ce1cd1961